samples/server/petstore/spring-mvc-j8-async/src/main/java/io/swagger/api/AnotherFakeApi.java

```diff
@@ -1,5 +1,5 @@
 /**
- * NOTE: This class is auto generated by the swagger code generator program (2.3.0-SNAPSHOT).
+ * NOTE: This class is auto generated by the swagger code generator program (unset).
```

Comment:
Weird...

Reply:
Hmm... there are somewhat more update scripts for linux than windows.  Are the windows ones maintained?  I'll try the linux ones...

Reply:
Yes, the windows scripts are broken